x/contractmanager/client/cli/query.go

```diff
@@ -0,0 +1,31 @@
+package cli
+
+import (
+	"fmt"
+	// "strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+
+	"github.com/neutron-org/neutron/x/contractmanager/types"
+)
+
+// GetQueryCmd returns the cli query commands for this module
+func GetQueryCmd(queryRoute string) *cobra.Command {
+	// Group contractmanager queries under a subcommand
+	cmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      fmt.Sprintf("Querying commands for the %s module", types.ModuleName),
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+
+	cmd.AddCommand(CmdQueryParams())
+	cmd.AddCommand(CmdAllFailures())
+	cmd.AddCommand(CmdContractFailures())
+	// this line is used by starport scaffolding # 1
```

Comment:
please remove all scaffolding mentions!

Reply:
It is useful to have them because `ignite` is using it to alter files (to add new messages, queries etc).